tests/pos/i11214.scala

```diff
@@ -0,0 +1,5 @@
+trait Pet(val name: String)
+trait FeatheredPet extends Pet
+
+class Bird(name: String) extends FeatheredPet:
```

Comment:
Why no “override val” is required here before `name`?

Reply:
I pushed the wrong version. The test does not pass.